save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/cosv/RawCosvFileDto.kt

```diff
@@ -33,4 +34,31 @@ data class RawCosvFileDto(
     fun requiredContentLength(): Long = requireNotNull(contentLength) {
         "contentLength is not provided: $this"
     }
+
+    companion object {
+        /**
+         * @return [Boolean]
+         */
+        fun RawCosvFileDto.isArchive() = status == RawCosvFileStatus.UPLOADED && fileName.endsWith(ARCHIVE_EXTENSION, ignoreCase = true)
+
+        /**
+         * @return [Boolean]
+         */
+        fun RawCosvFileDto.isJsonFile() = status == RawCosvFileStatus.UPLOADED && !fileName.endsWith(ARCHIVE_EXTENSION, ignoreCase = true)
```

Comment:
I thing !archive is not equals to the `json` file

Reply:
also, no need to check status

Reply:
Changed to `isUploadedZipArchive()` and `isUploadedJsonFile()`. The idea is to store in statistics only count of recently uploaded archives and json files to display info about it. For other types of `status` for files, different statistical information is displayed.